daemon/daemon.go

```diff
@@ -247,7 +247,7 @@ func (daemon *Daemon) restore() error {
 		}(c)
 	}
 	wg.Wait()
-	daemon.netController, err = daemon.initNetworkController(daemon.configStore, activeSandboxes)
+	daemon.netController, err = daemon.initNetworkController(daemon.configStore, daemon.pluginStore, activeSandboxes)
```

Comment:
You dont have to change the function signature here. Pls look for my next comment on how to implement it with the existing structure.